dali/base/dafdesc.cpp

```diff
@@ -3701,9 +3701,14 @@ class CStoragePlaneInfo : public CInterfaceOf<IStoragePlane>
     {
         return xml->queryProp("storageapi/@account");
     }
-    virtual const char * queryStorageContainer() const override
+    virtual StringBuffer & getStorageContainer(StringBuffer & container, unsigned stripeNumber) const override
```

Comment:
minor: curious why changed to fill in a StringBuffer, vs previous returning a pointer to the property value?